unitary/alpha/quantum_world.py

```diff
@@ -60,6 +67,7 @@ def clear(self):
         self.effect_history = []
         self.object_name_dict: Dict[str, QuantumObject] = {}
         self.ancilla_names = set()
+        self.compiled_qubits: Dict[cirq.Qid, List[cirq.Qid]] = {}
```

Comment:
Optional nit: consider a line comment to explain what this dict is.

Reply:
Done